packages/flame_forge2d/example/lib/main.dart

```diff
@@ -1,7 +1,26 @@
+import 'dart:html';
+
+import 'package:flame/game.dart';
+import 'package:flame_forge2d_example/joints/constant_volume_joint.dart';
+import 'package:flutter/widgets.dart';
+
 void main() {
-  // There will be a new example in here after Google I/O, stay tuned!
-  // If you want to see the previous examples, go to the flame_forge2d section
-  // of https://examples.flame-engine.org
-  // The source code lives here:
-  // https://github.com/flame-engine/flame/tree/main/examples/lib/stories/bridge_libraries/forge2d
+  var page = window.location.search ?? '';
+  if (page.startsWith('?')) {
+    page = page.substring(1);
```

Comment:
I would suggest just keeping one example inside flame_forge2d and then add multiple examples in here instead: https://github.com/flame-engine/flame/tree/main/examples/lib/stories/bridge_libraries

Reply:
Do you mean to have one example per joint type in flame_forge2d/example, and then more examples in stories/bridge_libraries, or leave one joint example here (does the one I added work?) and add the rest to stories_bridge_libraries?

Reply:
A more general example in flame_forge2d and then each joint type example can go in the big `examples` directory. (You don't have to create the general example though of you don't want to, it can just be something super simple like a ball bouncing around.)